src/Core/Grid/Definition/Factory/LogGridDefinitionFactory.php

```diff
@@ -49,6 +50,19 @@ final class LogGridDefinitionFactory extends AbstractGridDefinitionFactory
 {
     public const GRID_ID = 'logs';
 
+    /**
+     * @var string Date format for the current user
+     */
+    private $contextDateFormat;
+
+    public function __construct(
+        HookDispatcherInterface $hookDispatcher,
+        string $contextDateFormat
+    ) {
+        parent::__construct($hookDispatcher);
+        $this->contextDateFormat = $contextDateFormat;
```

Comment:
New constructor argument: BC break?

Reply:
Negative. The class is final so can't be inherited.

Reply:
I will note it in the rare case someone would instantiate this class manually.